internetarchive/cli/ia_upload.py

```diff
@@ -85,8 +86,23 @@
 )
 
 
+def _check_files(files):
+    return [
+        file
+        for file in files
+        if Path(file).exists()
+    ]
+
+
 def _upload_files(item, files, upload_kwargs, prev_identifier=None, archive_session=None):
     """Helper function for calling :meth:`Item.upload`"""
+    # Exclude files that do not exist.
+    files = _check_files(files)
+
+    # Check if the list has any element.
+    if len(files) < 1:
```

Comment:
Use the idiomatic `if not files:` instead.

Reply:
I am not really familiar with the whole project so I'm not sure if anything other than a list of strings can be passed to the function or not. That's why I did not trust truthy/falsy values. But since you mentioned it, there can't be a problem.